openmdao/components/tests/test_exec_comp.py

```diff
@@ -1685,7 +1686,10 @@ def test_register_check_partials_safe(self):
             assert_near_equal(p['comp.area_square'], np.ones(size) * 9., 1e-11)
 
             data = p.check_partials(out_stream=None)
-            self.assertEqual(list(data), [])
+            if env_truthy('GITHUB_ACTION') or env_truthy('TRAVIS'):
```

Comment:
I think we can just use the environment variable `CI` here..  it's boolean and available on GH Actions, Travis-CI and CircleCI:
https://docs.github.com/en/actions/learn-github-actions/environment-variables
https://docs.travis-ci.com/user/environment-variables
https://circleci.com/docs/env-vars

Reply:
fixed